eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -186,4 +203,53 @@ protected String variantToVcfLine(Long id, ISubmittedVariant variant) {
         return variantLine;
     }
 
+    /**
+     * Note that we can't use {@link ContigToGenbankReplacerProcessor} here because we allow other replacements than
+     * genbank, while that class is used to replace to genbank only (for writing in mongo and for comparing input and
+     * report VCFs).
+     */
+    private String getEquivalentContig(ISubmittedVariant normalizedVariant, ContigNaming contigNaming) {
+        String oldContig = normalizedVariant.getContig();
+
+        ContigSynonyms contigSynonyms = contigMapping.getContigSynonyms(oldContig);
+        if (contigSynonyms == null) {
+            if (!loggedUnreplaceableContigs.contains(oldContig)) {
+                loggedUnreplaceableContigs.add(oldContig);
+                logger.warn("Will not replace contig '" + oldContig
+                            + "' (in the current variant or any subsequent one) as requested because there are no "
+                            + "synonyms available. (Hint: Is the assembly report correct and complete?)");
+            }
+            return oldContig;
+        }
+
+        String replacedContig = contigSynonyms.get(contigNaming);
+        if (replacedContig == null) {
+            if (!loggedUnreplaceableContigs.contains(oldContig)) {
+                loggedUnreplaceableContigs.add(oldContig);
+                logger.warn("Will not replace contig '" + oldContig
+                            + "' (in the current variant or any subsequent one) as requested because there is no "
+                            + contigNaming + " synonym for it.");
+            }
+            return oldContig;
+        }
+
+        boolean genbankAndRefseq = oldContig.equals(contigSynonyms.getGenBank())
+                                   && replacedContig.equals(contigSynonyms.getRefSeq());
+
+        boolean refseqAndGenbank = oldContig.equals(contigSynonyms.getRefSeq())
```

Comment:
Suggest renaming the booleans to replacedWithRefSeq and replacedWithGenBank respectively.

Reply:
the important part is not that it's replaced to refseq, (or genbank), it's that it went from genbank to refseq (or from genbank to refseq).

I can use `genbankReplacedWithRefseq` and `refseqReplacedWithGenbank`